packs/feat-effects/effect-elemental-assault.json

```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Elemental Assault",
     "system": {
         "description": {
-            "value": "<p>You shroud your arms and held weapons in elemental magic. Choose one element. Until the end of your next turn, your Strikes deal an additional 1d6 damage of the indicated type and have the trait corresponding to the element: electricity for air, bludgeoning for earth, fire for fire, or cold for water.</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Elemental Assault]</p>\n<p>Your Strikes deal an additional 1d6 damage of the indicated type and have the trait corresponding to the element: electricity for air, bludgeoning for earth, fire for fire, or cold for water.</p>"
```

Comment:
Consider using `AdjustStrike` to add the trait to the strike.
See "Spell Effect: Inner Upheaval".